modules/ROOT/pages/UploadToSMR.adoc

```diff
@@ -56,15 +57,27 @@ or if you need to split your mod into multiple sub-mods.
 
 == UPlugin File
 
-Every unreal engine plugin, which satisfactory also are, require to have and
-`<mod reference>.uplugin` file.
+Every Unreal Engine plugin is require to have a
+`<mod reference>.uplugin` file,
+and Satisfactory mods are no exception.
 This file is json formatted and holds information about the mod like version, credits, links and more.
 
-You can read about all the different settings https://docs.unrealengine.com/en-US/API/Runtime/Projects/FPluginDescriptor/index.html[here]. But SML and SMR require you to have even more fields in your plugin descriptor file.
+You can read about all the different settings
+at the Plugins documentation
+https://docs.unrealengine.com/en-US/ProductionPipelines/Plugins/index.html#plugindescriptorfiles[here]
+or at the {cpp} reference
+https://docs.unrealengine.com/en-US/API/Runtime/Projects/FPluginDescriptor/index.html[here].
 
-Most importantly a mod dependency list and a SemVersion of the mod.
+If your mod has {cpp} code, make sure that you list your
+xref:Development/ModLoader/ModModules.adoc[Mod Modules]
+in the Modules plugin descriptor section.
+The examples below will demonstrate this.
```

Comment:
There is a difference between a UBT Module and a Mod Module... Mod Modules are a system introduced by SML, UBT Modules not, these an essentially chunks of native code and will get loaded by UE when listed in the modules section of the .uplugin file. A single UBT Module can even contain multiple Mod Modules...
So the point is, referring here to the Mod Modules is very misleading.